src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -110,13 +125,19 @@ def batch(iterable, chunks):
             mode=tiledb.cloud.dag.Mode.BATCH,
             resources=DEFAULT_RESOURCES if resources is None else resources,
             image_name=DEFAULT_IMG_NAME,
+            compressor=compressor_serial,
             **kwargs,
         )
 
     graph.compute()
     return graph
 
 
+def serialize_filter(filter):
+    if isinstance(filter, tiledb.Filter):
+        return {"type": type(filter).__name__, "attrs": filter._attrs_()}
```

Comment:
We could borrow [the filter specification syntax from TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA/tree/1.2.5/apis/python#:~:text=interface%20TDBFilterSpec%20%7B%0A%20%20/**%20The%20name%20of%20the%20filter.%20*/%0A%20%20_name%3A%20string%3B%0A%20%20/**%20kwargs%20that%20are%20passed%20when%20constructing%20the%20filter.%20*/%0A%20%20%5Bkwarg%3A%20string%5D%3A%20any%3B%0A%7D), which looks like:

```json
{"_name": "SomeFilter", "param1": 1, "param2": 2}
```

which would call `SomeFilter(param1=1, param2=2)`